lib/karafka/connection/client.rb

```diff
@@ -353,12 +362,14 @@ def events_poll(timeout = 0)
       # The offset field of each requested partition will either be set to stored offset or to
       # -1001 in case there was no stored offset for that partition.
       #
+      # @param [Rdkafka::Consumer::TopicPartitionList] tpl for which we want to get committed
```

Comment:
`# @param tpl [Rdkafka::Consumer::TopicPartitionList] for which we want to get committed`